src/query/block/types.go

```diff
@@ -37,6 +41,18 @@ type Block interface {
 	Close() error
 }
 
+// UnconsolidatedBlock represents a group of unconsolidated series across a time bound
+type UnconsolidatedBlock interface {
```

Comment:
Is there ever a scenario where you'd want to consolidate a single series rather than the entire block? If not, might be good to add a capability to do this at the block level

Reply:
Yeah, the UnconsolidatedBlock has a Consolidate() which does it at the block level. Not super sure what you mean.